app/models/work.rb

```diff
@@ -290,6 +290,11 @@ def self.successful_reindex(ids)
     Collection.expire_ids(collection_ids)
   end
 
+  # TODO: We can use touch_all once we update to Rails 6.
+  def touch_series
```

Comment:
Do we need to expire cache when the work is destroyed? If so, this method can be folded into `expire_caches`.

Reply:
No, just like removing a creator from a work doesn't update the series creators ([AO3-5739](https://otwarchive.atlassian.net/browse/AO3-5739)), neither does deleting a work.